src/combobox/styles/comboBox.css

```diff
@@ -0,0 +1,90 @@
+@import '../../common/styles/variables.css';
+
+.combobox {
+	position: relative;
+	display: inline-block;
+	box-sizing: border-box;
+	width: var(--form-field-width);
+}
+
+.combobox * {
+	box-sizing: border-box;
+}
+
+.input {
+	border: var(--border-thin) solid var(--dojo-shade-gray);
+	width: 100%;
+	font-size: var(--input-font-size);
+	font-family: var(--input-font-family);
+	padding: var(--input-padding);
+	padding-right: 25px;
+}
+
+.input.clearable {
+	padding-right: 44px;
+}
+
+.input:focus {
+	border-color: var(--dojo-border-focus);
+	box-shadow: var(--focus-highlight);
+	outline: none;
+}
+
+.results {
+	position: absolute;
+	width: 100%;
+	max-height: var(--select-menu-height);
+	overflow-y: auto;
+	background: var(--dojo-white);
+	border: var(--border-thin) solid var(--dojo-shade-gray);
+	z-index: 1;
+}
+
+.result {
+	cursor: pointer;
+	padding: var(--input-padding);
+}
+
+.result.disabledResult {
```

Comment:
no need to join these class names, or the `.input.clearable` selector above. simply styling `.clearable` should be fine.